src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -4,4 +4,5 @@ data class LottoNumber(
     val line: List<Int>
```

Comment:
목적에 따라서는 정말 자료구조 역할도 같이 가져가고 싶은 일급 컬렉션은 다음과 같이 위임을 통해 편의성을 높혀볼 수도 있습니다. 

```suggestion
    val line: List<Int>): List<Int> by line { ... }
```